lib/response.js

```diff
@@ -229,7 +236,39 @@ res.send = function send(body) {
     this.end();
   } else {
     // respond
-    this.end(chunk, encoding);
+    if (isChunkBlob) {
+      /**
+       * @type {http.ServerResponse}
+       */
+      var _this = this;
+      /**
+       * @type {Blob}
+       */
+      var blob = chunk;
+      try {
+        var WritableStream = require('stream/web').WritableStream;
```

Comment:
For any Node.js version where `require('stream/web')` doesn't work, this is going to cause a sync walk of the file system every time, since a `require()` failure is not added to the module cache like a success is. This would ideally be moved to only happen on the loading of this module, not on every single response write.